src/components/DayPicker.jsx

```diff
@@ -52,6 +54,8 @@ const defaultProps = {
   modifiers: {},
   orientation: HORIZONTAL_ORIENTATION,
   withPortal: false,
+  focused: true,
```

Comment:
absence and `false` should be the same - instead of a default of true, could this either default false, or use an inverted name such that it can default false?